routes/web.php

```diff
@@ -18,13 +18,18 @@
 Auth::routes();
 
 Route::get('/home', 'HomeController@index');
-Route::get('/projects/job-order/{id}/materials', 'Projects\\JobOrderController@materials');
+
+Route::get('projects/job-order/{id}/materials/create', 'Projects\\MaterialsController@create');
+Route::get('projects/job-order/{id}/materials/{matid}', 'Projects\\MaterialsController@update');
+Route::get('projects/job-order/{id}/materials/{matid}', 'Projects\\MaterialsController@destroy');
+Route::get('projects/job-order/{id}/materials/{matid}/show', 'Projects\\MaterialsController@show');
+Route::get('projects/job-order/{id}/materials/{matid}/edit', 'Projects\\MaterialsController@edit');
```

Comment:
This is needed in order to get both the job_id and mat_id which is used on each respective functions.